pages/endpoint/workflow.mdx

```diff
@@ -0,0 +1,143 @@
+import { OptionTable } from '@components/table'
+
+# Workflow
+
+The Workflow class provides methods for executing and managing workflow tasks, such as image upscaling.
+
+## Execute Workflow and Wait for Result
+
+Executes a workflow task and waits for the result.
+
+### Input
+
+```ts {8-14} copy
+import Heurist from 'heurist'
+
+const heurist = new Heurist({
+  apiKey: process.env['HEURIST_API_KEY'], // This is the default and can be omitted
+})
+
+async function main() {
+  const upscalerTask = new UpscalerTask({
+    consumer_id: 'example-id',
+    workflow_id: '1',
+    image_url: 'https://example.com/sample.jpg'
+  });
+  const response = await heurist.workflow.executeWorkflowAndWaitForResult(upscalerTask);
+}
+
+main()
+```
+
+### Response
+
+```json
+{
+  "task_id": "example-task-id",
+  "status": "finished",
+  "result": "upscaled-image.png"
+}
+```
+
+## Parameters
+
+- Type: `UpscalerTask`
+
+<OptionTable
+  language="en"
+  options={[
+    [
+      'consumer_id',
+      'string',
+      true,
+      'The ID of the consumer initiating the task.',
```

Comment:
What's the concept and requirement of consumer_id? can add more description?

Reply:
updated